static/js/modules/send-message.js

```diff
@@ -150,26 +151,26 @@ var _ = require('underscore'),
     $modal.find('.has-error').removeClass('has-error');
     $modal.find('.help-block').text('');
 
-    var val = [],
+    var initialTo,
         to = options.to,
         message = options.message || '';
 
     if (to) {
       if (typeof to === 'string') {
-        val.push(to);
+        initialTo = [to];
```

Comment:
It'd be fine just to modify `to` and not have `initialTo` at all. I presume this was a stylistic choice?